src/components/inputs/async-creatable-select-input/async-creatable-select-input.js

```diff
@@ -2,10 +2,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { injectIntl } from 'react-intl';
 import { withTheme } from 'emotion-theming';
-import {
-  components as defaultComponents,
-  AsyncCreatable as AsyncCreatableSelect,
-} from 'react-select';
+import { components as defaultComponents } from 'react-select';
+import AsyncCreatableSelect from 'react-select/async-creatable';
```

Comment:
Are they doing this on purpose to have better treeshaking?

Reply:
Yeah 

this is what they said

"This should have no bundle-size impact on react-select consumers currently leveraging tree-shaking. However for consumers who aren’t leveraging tree-shaking, this should help alleviate some of the bundle-weight."